src/nvim/ops.c

```diff
@@ -5404,9 +5418,44 @@ static bool get_clipboard(int name, yankreg_T **target, bool quiet)
   return false;
 }
 
+
+/*
```

Comment:
could merge this two comment blocks and into doxygen style. Also I would suggest moving `start_`/`end_` below `set_clipboard` ( as `adjust_clipboard_name`, `get_clipboard`, and `set_clipboard` belong close together, while `*_global_changes` is more of a special case)